From rpjday@crashcourse.ca  Fri Mar  7 09:03:59 2008
From: "Robert P. J. Day" <rpjday@crashcourse.ca>
Date: Fri, 7 Mar 2008 11:40:07 -0500 (EST)
Subject: USB: Remove __KERNEL__ check from non-exported gadget.h.
To: linux-usb@vger.kernel.org
Message-ID: <alpine.LFD.1.00.0803071138270.28191@localhost.localdomain>



Since the header file gadget.h isn't being exported to userspace,
there seems to be little point having a __KERNEL__ proprocessor check.

Signed-off-by: Robert P. J. Day <rpjday@crashcourse.ca>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/usb/gadget.h |    4 ----
 1 file changed, 4 deletions(-)

--- a/include/linux/usb/gadget.h
+++ b/include/linux/usb/gadget.h
@@ -15,8 +15,6 @@
 #ifndef __LINUX_USB_GADGET_H
 #define __LINUX_USB_GADGET_H
 
-#ifdef __KERNEL__
-
 struct usb_ep;
 
 /**
@@ -848,6 +846,4 @@ extern struct usb_ep *usb_ep_autoconfig(
 
 extern void usb_ep_autoconfig_reset(struct usb_gadget *) __devinit;
 
-#endif  /* __KERNEL__ */
-
 #endif	/* __LINUX_USB_GADGET_H */
